src/Uno.UWP/Devices/Sensors/SimpleOrientationSensor.iOS.cs

```diff
@@ -8,20 +9,40 @@ namespace Windows.Devices.Sensors
 	public partial class SimpleOrientationSensor
 	{
 		private SimpleOrientation _previousOrientation;
-		private static CMMotionManager _motionManager;
+		private static CMMotionManager? _motionManager;
 		private const double _updateInterval = 0.5;
 		private const double _threshold = 0.5;
 
 		partial void Initialize()
 		{
+			if (_motionManager is { DeviceMotionAvailable: true })
+			{
+				// We should not start more than one CMMotionManager to avoid performance hit.
+				_motionManager.StopDeviceMotionUpdates();
```

Comment:
In theory, we should not have multiple instances of this class (it's a singleton) but I wonder, how did you fall in this case?